ashley/src/main/kotlin/ktx/ashley/componentMappers.kt

```diff
@@ -0,0 +1,17 @@
+package ktx.ashley
+
+import com.badlogic.ashley.core.Component
+import com.badlogic.ashley.core.ComponentMapper
+
+/**
+ * Creates a [ComponentMapper] for the specified [Component] type.
+ *
+ * Provides `O(1)` retrieval of [Component]s for an [com.badlogic.ashley.core.Entity].
+ *
+ * @param T the [Component] type to create a [ComponentMapper] for
+ * @return the [ComponentMapper]
+ * @see ComponentMapper
+ */
+inline fun <reified T: Component> mapperFor(): ComponentMapper<T> {
+  return ComponentMapper.getFor(T::class.java)
```

Comment:
I know that it comes down to preferences, but I consider `=` syntax to be more readable than `{ return ... }` when it comes to short, single-line methods.

```Kotlin
inline fun <reified T: Component> mapperFor(): ComponentMapper<T> = ComponentMapper.getFor(T::class.java)
```